### PR TITLE
IAM integration

### PR DESCRIPTION
**Pull request by johngian**
* Add OIDC authentication support
* Use ``django-environ`` for ``settings/local_settings.py``.
* Minor changes for local setup

### CODE REVIEW

**Comment by johngian** [comment]
Fixes #109

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length=99
+exclude=migrations
```
Comment (akatsoulas) [inline setup.cfg]: I would suggest to keep the PRs as clean as possible. 👍 for all the commits but I would suggest to open different PRs for flake/setup.cfg and env vars.

**Comment by Pomax** [comment]
Also, if I should be reviewing anything here, can you add me as reviewer so it shows up in my queues?

File: setup.cfg (hunk 1 above)
Comment (johngian) [inline setup.cfg]: Commits are kinda atomic so I think that its not that bad for now. I would make sure i split it in future PRs.

**Pull request merged**